odra-casper/wasm-env/src/lib.rs

```diff
@@ -21,10 +21,10 @@ pub use wasm_contract_env::WasmContractEnv;
 #[allow(unused_imports)]
 use ink_allocator;
 
-/// Panic handler for the WASM target architecture.
-#[cfg(target_arch = "wasm32")]
-#[panic_handler]
-#[no_mangle]
-pub fn panic(_info: &core::panic::PanicInfo) -> ! {
-    core::intrinsics::abort();
-}
+// /// Panic handler for the WASM target architecture.
```

Comment:
What happed with the panic handler?

Reply:
It is implemented by one of the casper crates already.